lua/conform/util.lua

```diff
@@ -246,4 +248,60 @@ M.shell_build_argv = function(cmd)
   return argv
 end
 
+---@param file string
+---@return nil|table
+local function read_json(file)
+  local f = io.open(file, "r")
+  if not f then
+    log.error("Unable to open file %s", file)
+    return nil
+  end
+
+  local file_content = f:read("*all") -- Read entire file contents
+  f:close()
+
+  local ok, json = pcall(vim.json.decode, file_content)
+  if not ok then
+    log.error("Unable to parse json file %s", file)
+    return nil
+  end
+
+  return json
+end
+
+-- @param self conform.FormatterConfig
+-- @param ctx conform.Context|conform.RangeContext
+-- @return string[]|nil args the arguments for setting a `prettier` parser if they exist in the options, nil otherwise
+M.prettier_cwd = function(self, ctx)
```

Comment:
I have a gut feeling it should be a part of some external dependency, so it could be shared between other plugins like nvim-lint, etc. Until then this is only used by 2 formatters, so this does not seem to be the best to share it via a general utils file that is used everywhere. I'm considering to move this into the `lua/conform/util/prettier-cwd` or smth like that now.